### ISSUE TITLE
Message schemas do not include releases

### ISSUE THREAD

**Issue by AdamWill** [fedora-infra/bodhi#3456]
Maybe I'm missing something here, but I can't find anywhere that the message schemas for update messages require that the message include the release dict for the release that the update is for.

The published messages *do* include the `release` dict inside the `update` dict at least at present, but the schemas are supposed to be the authoritative reference to what you can rely on being present in the messages, anything not in the schema could theoretically get taken out. So I think the release dict should be in the schemas...

**Comment by AdamWill** [comment]
Another thing I'm curious about: `content_type` property of the `update` object is not in the schema either. Not sure if that's an intentional omission, it seems like important information.

**Comment by bowlofeggs** [comment]
Yeah these things were purposefully omitted as documented here:

https://github.com/fedora-infra/bodhi/blame/9b440bc7d7db882c3adada6236e8d5652a69530e/docs/user/release_notes.rst#L186

The idea is that the messages that Bodhi sends now are highly suboptimal, mostly in that they serialize way more data than is likely needed, but we also didn't have the time when writing the schemas to change the messages too much. So I opted instead to only commit to the documented attributes.

If you need to use some data that isn't in the schema, we can just add it to the schema (after potentially making it more efficiently represented if feasible).

tl;dr; we only want to commit to things in the schema that we want to continue to support, and we should think a bit before just adding everything in the current message to the schema.

**Comment by bowlofeggs** [comment]
Can you name the fields from the release that you want to include in the schema?

**Comment by AdamWill** [comment]
Well, I mean, the goal is 'reliably identify the release this update is for', I guess. You know better than I do which fields would be necessary for that purpose. At a guess, the most important field would be `name`, assuming that can always be relied on as a unique identifier that will be easily queryable? Maybe `name` and `version`...

**Comment by ryanlerch** [comment]
The release is now included in the schema for updates:

https://github.com/fedora-infra/bodhi/pull/3532
